addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -348,3 +348,43 @@ registry.category("web_tour.tours").add("PosPopupPriceAndQuantity", {
             },
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosCategoriesOrder", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            {
+                content: `Check Categories Order`,
+                trigger: '.category-button:eq(0) > span:contains("AAA")',
+                timeout: 5000000,
```

Comment:
don't use timeout. also, you can remove all the 'content' field values.

Reply:
Woow sorry by mistake.